### DISCUSSION TITLE
[firebase_dynamic_links]  Firebase Dynamic Links is deprecated

### DISCUSSION THREAD

**Discussion by mdmota** [firebase/flutterfire#12155] [Q&A]
With the discontinuation of firebase dynamic links, will the plugin also stop working or can I continue using it to capture the link?

**Comment by Vaytrex** [comment]
For anyone landing here after the Aug 2025 shutdown: there's no drop-in. You need a URL that branches on User-Agent (App Store / intent:// with browser_fallback_url / website) and skips redirecting crawlers so previews survive. Full write-up with the intent format: https://linkbah.app/firebase-dynamic-links-alternative
